### PR TITLE
Game Support: Add GTA IV

### PR DESCRIPTION
**Pull request by felix-schindler**

### CODE REVIEW

**Review by mattmaniak — CHANGES_REQUESTED** [review]
Please run the "generate" script to add the game to games list.

File: src/game-support/gta-4.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "GTA IV",
+    "Grand Theft Auto 4"
+] -->
+
+{{#template ../templates/rating.md status=Garbage installs=Yes opens=No}}
+
+> [!WARNING]
+> The Rockstar Games Launcher will fail with the `The Rockstar Games Launcher failed to initialize!` error when `DXVK` is turned `off` in your `Bottle Configuration`. `DXVK` can turned `on` to workaround the launcher error.
+
+Crashes during load without error message and is therefore currently unplayable in Whisky.
```
Comment (mattmaniak) [inline src/game-support/gta-4.md:13]: Does the game crashes itself or the Rockstar Launcher?

Comment (mattmaniak) [reply src/game-support/gta-4.md:13]: Additionally please add Steam ID of GTA IV as it may be helpful for others to locate this game on Steam.

**Comment by felix-schindler** [comment]
superseded by #105

**Pull request closed without merging**